### PR TITLE
More themes

### PR DESCRIPTION
**Pull request by shadmansaleh**
Added themes:
- Tomorrow.lua
- ayu_dark.lua
- ayu_light.lua
- ayu_mirage.lua
- iceberg_dark.lua
- iceberg_light.lua
- jellybeans.lua
- oceanicnext.lua
- onehalf.lua
- seoul256.lua

I'm done with themes . I think I've got most of what we need.

### CODE REVIEW

File: THEMES.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
 
 All available themes are only best effort ports by myself/ other users. If you find a theme to be weird/ wrong please open an issue/ pr.
 
+### ayu dark
```
Comment (hoob3rt) [inline THEMES.md:5]: all names should be changed to work directly in lualine. 
```vim
:lua require('lualine').theme = 'ayu dark'
```
Obviously breaks, should be `ayu_dark` in THEMES.md

Comment (shadmansaleh) [reply THEMES.md]: fixed

**Comment by hoob3rt** [comment]
`onehalf` looks like just another `onedark` implementation. We don't need it if #70 is merged

**Comment by shadmansaleh** [comment]
> onehalf looks like just another onedark implementation. We don't need it if #70 is merged

Yes it seems both are quite similar. Though One Half doesn't state it's a derivation of onedark their statusline implementation seems same . It can be removed.

**Comment by hoob3rt** [comment]
some improvements needed:
ayu_light and ayu_mirage blend with background, this is bad imo

**Comment by shadmansaleh** [comment]
How about now?

**Comment by hoob3rt** [comment]
I fixed it myself, looks better now imo

**Pull request merged**